docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -483,14 +483,20 @@ This feature is available in Enterprise Edition only.
    > Please note that the images will be uploaded to the registry along the path specified in the `--registry` parameter (in the example above - /deckhouse/ee).
    > Before running the command, make sure this path exists in your registry, and the account you are using has write permissions.
 
-   If your registry does not require authentication, omit both `--registry-login` and `--registry-password` flags.
+   If your registry does not require authentication, omit both `--registry-login` and `--registry-password` flags or `DHCTL_CLI_MIRROR_USER`/`DHCTL_CLI_MIRROR_PASS` variables.
```

Comment:
```suggestion
   If your registry does not require authentication, you may omit both `--registry-login` and `--registry-password` flags as well as `DHCTL_CLI_MIRROR_USER`/`DHCTL_CLI_MIRROR_PASS` variables.
```

Reply:
"Or" in this case may be interpreted as "omit the variables but specify the flags" or vice versa. What we mean to say is that specifying them is not necessary, right?

Reply:
Exactly